content/docs/command-reference/stage/add.md

```diff
@@ -0,0 +1,432 @@
+# stage add
+
+Helper command to add or update <abbr>stages</abbr> in `dvc.yaml`. Requires a
+name and a command.
+
+## Synopsis
+
+```usage
+usage: dvc stage add [-h] [-q | -v] -n <name> [-d <path>] [-o <path>]
+                     [-O <path>] [-p [<path>:]<params_list>] [-m <path>]
+                     [-M <path>] [--plots <path>] [--plots-no-cache <path>]
+                     [-w <path>] [-f]
+                     [--outs-persist <path>] [--outs-persist-no-cache <path>]
+                     [--always-changed] [--external] [--desc <text>]
+                    command
```

Comment:
If I keep the `command` indented, the syntax highlighting fails. If I add one space or remove it, it mysteriously works.

Reply:
it's probably a bug as well - mind creating a ticket? with this one and the previous.

Reply:
I tried this locally (added a space before `command` to align it) and the commit checks passed. Can you clarify please? 🙂

Reply:
Ah I see you were talking about syntax highlighting. Nvmd